src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    protected String task;
+    protected boolean isDone;
+
+    public Task (String task) {
+        this.task = task;
+        this.isDone = false;
+    }
+
+    public void markDone() {
+        this.isDone = !isDone;
+    }
+
+    public String getTaskType() {
+        return "";
+    }
+
+    public String getDeadline() {
+        return "";
+    }
+
+    public String getStart() {
+        return "";
+    }
+
+    public String getEnd() {
+        return "";
+    }
+
+    public void iji() {
```

Comment:
what exactly is this function supposed to mean? the naming is very unclear. 
https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#use-standard-words